src/nimble.nim

```diff
@@ -355,6 +356,18 @@ proc allDependencies(pkgInfo: PackageInfo, options: Options): HashSet[PackageInf
   for requires in pkgInfo.taskRequires.values:
     result.incl pkgInfo.processFreeDependencies(requires, options)
 
+proc isSubdirOf(subdir, baseDir: string): bool =
+  subDir.normalizedPath.startsWith(baseDir.normalizedPath)
```

Comment:
This is only correct if normalizedPath ends in `/`!

Reply:
I dont understand, `normalizedPath` get rids of `/` i.e. `/a/b/c/` becomes `/a/b/c`